salt/modules/htpasswd.py

```diff
@@ -73,7 +73,7 @@ def useradd(pwfile, user, password, opts=''):
         salt '*' webutil.useradd /etc/httpd/htpasswd larry badpassword
         salt '*' webutil.useradd /etc/httpd/htpasswd larry badpass opts=ns
     '''
-    return useradd_all(pwfile, user, password, opts=opts)['out'].splitlines()
+    return useradd_all(pwfile, user, password, opts=opts)
```

Comment:
The line should be:

```
return useradd_all(pwfile, user, password, opts=opts)['stdout'].splitlines()
```

Can you please check that this line works and fix accordingly?